<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>599</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Expt-o Morris</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabViews">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>14</y>
      <width>1001</width>
      <height>536</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border: 1px solid #C2C7CB;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gravação</string>
     </attribute>
     <widget class="QLabel" name="lblViewGravacao">
      <property name="geometry">
       <rect>
        <x>487</x>
        <y>17</y>
        <width>424</width>
        <height>424</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGravar">
      <property name="geometry">
       <rect>
        <x>771</x>
        <y>446</y>
        <width>141</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gravar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFPS">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>447</y>
        <width>270</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FPS</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="cboTimerLimite">
      <property name="geometry">
       <rect>
        <x>293</x>
        <y>345</y>
        <width>74</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="displayFormat">
       <string>mm:ss</string>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>10</second>
       </time>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpFormato">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>102</y>
        <width>348</width>
        <height>115</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Formato</string>
      </property>
      <widget class="QRadioButton" name="rdb2Dcor">
       <property name="geometry">
        <rect>
         <x>239</x>
         <y>29</y>
         <width>88</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2D (cor)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdb2Dinfra">
       <property name="geometry">
        <rect>
         <x>133</x>
         <y>28</y>
         <width>84</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2D (IR)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdb3Dcor">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>70</y>
         <width>89</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3D (cor)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdb3Dcinza">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>69</y>
         <width>105</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3D (cinza)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdb2Dcinza">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>27</y>
         <width>105</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2D (cinza)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="txtId">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>49</y>
        <width>325</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblFPSIR_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>107</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Identificação</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>39</y>
        <width>348</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkTrack">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>379</y>
        <width>180</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rastrear na gravação</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkPararAuto">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>343</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Parar gravação automaticante</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdDisplay">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>230</y>
        <width>348</width>
        <height>89</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <zorder>textEdit</zorder>
     <zorder>lblViewGravacao</zorder>
     <zorder>btnGravar</zorder>
     <zorder>lblFPS</zorder>
     <zorder>cboTimerLimite</zorder>
     <zorder>grpFormato</zorder>
     <zorder>txtId</zorder>
     <zorder>lblFPSIR_2</zorder>
     <zorder>chkTrack</zorder>
     <zorder>chkPararAuto</zorder>
     <zorder>lcdDisplay</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Visualização</string>
     </attribute>
     <widget class="QPushButton" name="btnVisualizar">
      <property name="geometry">
       <rect>
        <x>825</x>
        <y>455</y>
        <width>99</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Visualizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblViewTracks">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>29</y>
        <width>424</width>
        <height>424</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>424</width>
        <height>424</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
